addons/service/multimedia/streamdisc/changelog.txt

```diff
@@ -0,0 +1,2 @@
+0.1
```

Comment:
Does that mean The addon wont be available for OE6 Users ?

Von meinem Mobiltelefon gesendet

> Am 20.01.2016 um 20:34 schrieb Lukas Rusak <notifications@github.com>:
> 
> In addons/service/multimedia/streamdisc/changelog.txt:
> 
> > @@ -0,0 +1,2 @@
> > +0.1
> should be 7.0.0 now
> 
> —
> Reply to this email directly or view it on GitHub.
>